### ISSUE TITLE
LXD/ARM: Handle correctly stop of Ephemeral container and task

### ISSUE THREAD

**Issue by geaaru** [MottainaiCI/mottainai-server#41]
It seems that at the end of a very long ephemeral container the call to stop container is not closed correctly and the task remains stalled.

Not sure if a Mottainai issue but it seems an LXD/LXC issue related to LXD+ZFS+ARM.

Jan 27 14:36:28 jarvis mottainai-agent[2603]: time="2019-01-27T14:36:28+01:00" level=info msg="2019/01/27 14:36:28 [**Trying to execute action stop to container mottainai-sabayon-builder-arm-3098133925860599387**: &{{armv7l map[volatile.last_state.idmap:[{\"Isuid\":true,\"Isgid\":false,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":65536},{\"Isuid\":false,\"Isgid\":true,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":65536}] volatile.mottainai.hwaddr:00:16:3e:18:a0:42 image.architecture:armhf image.name:sabayon--armhf-builder-20190120_1339 image.os:sabayon image.release: image.serial:20190120_1339 volatile.base_image:c2ff33757e7ef6bd97b4b3b3a9300f9e37a6a7525d47f64a13a2c4b39cc86ef6 image.description:Sabayon  armhf (builder) (20190120_1339) image.variant:builder volatile.idmap.base:0 volatile.idmap.next:[{\"Isuid\":true,\"Isgid\":false,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":65536},{\"Isuid\":false,\"Isgid\":true,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":65536}] volatile.last_state.power:RUNNING] map[] true [root-zfs-data2 net-mottainai]  false } 2019-01-25 08:52:56.658569666 +0100 CET map[image.serial:20190120_1339 volatile.base_image:c2ff33757e7ef6bd97b4b3b3a9300f9e37a6a7525d47f64a13a2c4b39cc86ef6 volatile.idmap.next:[{\"Isuid\":true,\"Isgid\":false,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":65536},{\"Isuid\":false,\"Isgid\":true,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":65536}] image.name:sabayon--armhf-builder-20190120_1339 image.description:Sabayon  armhf (builder) (20190120_1339) image.os:sabayon image.release: image.variant:builder volatile.idmap.base:0 volatile.last_state.idmap:[{\"Isuid\":true,\"Isgid\":false,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":65536},{\"Isuid\":false,\"Isgid\":true,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":65536}] volatile.last_state.power:RUNNING image.architecture:armhf volatile.mottainai.hwaddr:00:16:3e:18:a0:42] map[mottainai:map[nictype:bridged parent:mottainai type:nic name:mottainai] root:map[path:/ pool:zfs-data2 type:disk]] mottainai-sabayon-builder-arm-3098133925860599387 Running Running 2019-01-25 08:56:17.570099863 +0100 CET none}]"

related to this row: 

https://github.com/MottainaiCI/mottainai-server/blob/master/pkg/tasks/lxd.go#L613

**Comment by geaaru** [comment]
Not sure if could be related to this:

```
# cat lxc.log
lxc mottainai-sabayon-builder-arm-3098133925860599387 20190125075618.499 WARN     cgfsng - cgroups/cgfsng.c:chowmod:1556 - No such file or directory - Failed to chown(/sys/fs/cgroup/unified//lxc.payload/mottainai-sabayon-builder-arm-3098133925860599387/cgroup.threads, 65536, 0)
lxc mottainai-sabayon-builder-arm-3098133925860599387 20190125075618.894 WARN     conf - conf.c:lxc_setup_devpts:1641 - Invalid argument - Failed to unmount old devpts instance
lxc mottainai-sabayon-builder-arm-3098133925860599387 20190127133633.379 WARN     network - network.c:lxc_delete_network_priv:2589 - Operation not permitted - Failed to remove interface "mottainai" with index 19
```

**Comment by geaaru** [comment]
Fixed [3fa0f6f4c52dab61844d04c102b0606880333c3a]
